### ISSUE TITLE
Keine Rückkehr zur Trefferliste bei Bearbeitung von Buchungen

### ISSUE THREAD

**Issue by wklingberg** [tkrebs/ep3-bs#93]
"Bearbeitet" man aus der Trefferliste eine Buchung, landet man bei jeder Folgeaktion auf der "Kalender-Seite" und nicht auf der vorhergehenden Trefferliste.
Schon beim Aufruf einer Buchung gibt es nur den Navigationsbutton "Zurück zu Kalender".

**Comment by Stefan-Ha** [comment]
ja, ist nervig. Insbesondere wenn man nach einer Testphase die ganzen Testbuchungen löschen möchte. Ich denke, dass man es über den Zugriff auf die MySql Datenbank besser hinkriegt.
